src/app.js

```diff
@@ -1 +1,25 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const createFile = () => {
+  const date = new Date();
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const seconds = date.getSeconds();
```

Comment:
The time format should be `17:05:25` as the standard one. But in your case, it's: `17:5:25`.
If there is a digit < `10` , there should be the `0` before.